### PR TITLE
Zaz

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://sourceforge.net/p/zaz/discussion/996643/thread/b803210f/
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://sourceforge.net/p/zaz/discussion/996643/thread/b803210f/

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build io.sourceforge.zaz

**Comment by Mailaender** [comment]
bot, build io.sourceforge.zaz

File: io.sourceforge.zaz.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  - name: sdl12-compat
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: https://github.com/libsdl-org/sdl12-compat/archive/refs/tags/release-1.2.52.tar.gz
+        sha256: 5bd7942703575554670a8767ae030f7921a0ac3c5e2fd173a537b7c7a8599014
+
+  - shared-modules/SDL/SDL_image-1.2.12.json
+
+  - name: ftgl
```
Comment (hfiguiere) [inline io.sourceforge.zaz.yaml:28]: you never cleanup include and other development specific files.

Same with sdl12

**Comment by Mailaender** [comment]
bot, build io.sourceforge.zaz

**Pull request closed without merging**